UnitTests/TraitsMock.h

```diff
@@ -12,16 +12,19 @@ namespace UnitTest
 {
 namespace Mocks
 {
-    struct Traits : DefaultTraits
+    struct NullTimeoutTraits : DefaultTraits

```

Comment:
>Null [](start = 11, length = 4)

I'd probably name it "Infinite", if this policy will cause timeout to never happen.